src/multibody/geometry.hpp

```diff
@@ -274,100 +268,61 @@ struct GeometryObject
 
     GeometryModel(const se3::Model & model)
       : model(model)
-      , ncollisions(0)
-      , nvisuals(0)
-      , collision_objects()
-      , visual_objects()
+      , ngeoms(0)
+      , geometry_objects()
```

Comment:
What I dont like is that "object" is not meaningfull. Typically, GeometryModel is in a sense a geometry object.
Let's keep it like that for now, then, since no nice idea of name emerges. Shouldnt it be geometryObject to fit with the conventions in Pinocchio?